### ISSUE TITLE
Bootstrap netshell-root problem

### ISSUE THREAD

**Issue by bmah888** [esnet/enos#78]
The activators for netshell-kernel and netshell-python seem very confused when running as OSGi bundles.  It looks like they're not able to correctly figure out the correct netshell-root directory (maybe because the configuration file wasn't read?).

```
2016-04-21 21:10:45,561 | INFO  | Thread-64        | NetShellConfiguration            | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | Master configuration file is /home/bmah/distro/netshell.conf
2016-04-21 21:10:45,654 | WARN  | Thread-64        | KernelSecurityManager            | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | NetShell SecurityManager is currently disabled.  No security checks will be run.  MUST NOT BE USED IN PRODUCTION.
2016-04-21 21:10:45,654 | INFO  | Thread-64        | BootStrap                        | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | Starting BootStrap thread
2016-04-21 21:10:45,654 | INFO  | Thread-64        | BootStrap                        | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | Current directory: /home/bmah/distro/distribution-karaf-0.3.1-Lithium-SR1
2016-04-21 21:10:45,654 | INFO  | tShell Bootstrap | BootStrap                        | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | Starting services
2016-04-21 21:10:45,657 | ERROR | Thread-64        | BootStrap                        | 399 - net.es.netshell-kernel - 1.0.0.SNAPSHOT | NetShell root directory / not writable
```

Going to take a look at some changes to the Bootstrap class to see if that might be a factor.  This was observed in a freshly-compiled ODL Karaf on a new LXC.

**Comment by lomaxfrog** [comment]
Problem was due to a race condition happening only on very slow host.

Bruce could you please verify and close this issue if it is fixed.

**Comment by bmah888** [comment]
Fixed by esnet/netshell@b7d6ff78fc3a8ebe8cfdfa7a3e51ce35201709fc.

Closing after brief testing.
